src/utils/cookieUtils.ts

```diff
@@ -0,0 +1,20 @@
+'use server'
+import { cookies } from 'next/headers'
+
+interface CookiesSetOptions {
+  [key: string]: string
+}
+
+// eslint-disable-next-line @typescript-eslint/require-await
+export async function setCookie(
```

Comment:
What is the purpose of making this function `async`?

Reply:
The server-side environment of the Next.Js framework requires that functions must be asynchronous only and not otherwise! Since cookies().set(key, value, options) and cookies().set(key, '', { maxAge: 0 })
 I can't execute with 'await' I had to use the linter comment